server/the-usher-openapi-spec.yaml

```diff
@@ -469,6 +469,29 @@ paths:
         500:
           $ref: '#/components/responses/InternalError'
 
+  /personas/{persona_key}:
+    delete:
+      'x-swagger-router-controller': 'personas/persona'
+      operationId: deletePersona
+      summary: Deletes the subject persona
+      parameters:
+      - $ref: '#/components/parameters/personaKeyPathParam'
+      tags:
+        - Admin APIs
+      security:
+        - bearerAdminAuth: []
+      responses:
+        204:
+          description: Empty response on success
+        401:
```

Comment:
@k1ch small nit, should the response codes be listed in ascending order? If you want to fix then also make the change for the spec added in #89 .

Reply:
Yes, it should. I will update.